### PR TITLE
Typos Pass + Go Mod Tidy

### PR DESCRIPTION
**Pull request by SyntaxNode**
I have a spell checked in my VS Code setup. Fixing some typos that were discovered.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,4 +2,4 @@ module github.com/prebid/go-gpp
 
 go 1.16
 
-require github.com/stretchr/testify v1.8.1 // indirect
+require github.com/stretchr/testify v1.8.1
```
Comment (SyntaxNode) [inline go.mod:5]: This is actually used directly by tests. Ran `go mod tidy` to fix the annotation.

**Review by camrice — APPROVED** [review]

File: util/bitstream.go [hunk 2]
```diff
@@ -11,7 +11,7 @@ type BitStream struct {
 	b []byte
 }
 
-// NewBitStream creates a new butstream object
```
Comment (hhhjort) [inline util/bitstream.go:14]: Gotta love the butstream

**Review by hhhjort — APPROVED** [review]
LGTM

File: util/bitstream.go (hunk 2 above)
Comment (SyntaxNode) [reply util/bitstream.go:14]: 😂

**Pull request merged**